app.js

```diff
@@ -23,3 +24,31 @@ app.get('/', (req, res) => {
 });
 
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
+
+app.get("/", (req, res) => {
```

Comment:
You didn't need this route in here! There's one right before it that is making it work